io.github.nate_xyz.Conjure.json

```
@@ -0,0 +1,50 @@
+{
+    "app-id" : "io.github.nate_xyz.Conjure",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "conjure",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "magick",
+            "buildsystem": "autotools",
+            "config-opts": "--prefix=/app --with-heic=yes --with-rsvg=yes",
```

```suggestion
            "config-opts": [
                "--with-heic=yes",
                "--with-rsvg=yes"
            ],
```

Note that flatpak-builder throw this warning:

`(flatpak-builder:2): Json-WARNING **: 16:07:30.774: Failed to deserialize "config-opts" property of type "GStrv" for an object of type "BuilderModule"`

The documentation clearly say that it's an array of string.

---

since prefix is set automatically it did build, but without the options you want.

Note that heic is not in the runtime, or at least not detected by ImageMagick

---

Thanks, I must have missed that warning.